### PR TITLE
pretty-printing API fix

### PR DESCRIPTION
**Pull request by yaskevich**
Pretty-printing API unified with that of `pino`/`pino-pretty`, as it is presented [here](https://getpino.io/#/docs/pretty).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -87,8 +87,8 @@ function pinoMultiStream (opts, stream) {
 Object.assign(pinoMultiStream, pino)
 pinoMultiStream.multistream = multistream
 pinoMultiStream.prettyStream = (args = {}) => {
-  const { opts = {}, prettifier, dest = process.stdout } = args
-  return getPrettyStream(opts, prettifier, dest)
+  const { prettyPrint = {}, prettifier, dest = process.stdout } = args
```
Comment (jsumners) [inline index.js]: This is a breaking change.

Comment (mcollina) [reply index.js]: Can you make it a non-breaking change? You'll need to also support `opts`.

Comment (yaskevich) [reply index.js]: > Can you make it a non-breaking change? You'll need to also support `opts`.
Right. I've updated the PR.

File: test/wrapper.test.js [hunk 2]
```diff
@@ -303,8 +303,8 @@ test('creates pretty write stream with custom options for pino-pretty', function
       t.done()
     }
   })
-  const opts = { colorize: false, ignore: 'hostname,pid,time' }
-  const prettyStream = pinoms.prettyStream({ opts, dest })
+  const prettyPrint = { colorize: false, ignore: 'hostname,pid,time' }
+  const prettyStream = pinoms.prettyStream({ prettyPrint, dest })
```
Comment (mcollina) [inline test/wrapper.test.js]: Can you add a test for the `opts` option?

**Comment by yaskevich** [comment]
I've added tests for old option.

**Comment by mcollina** [comment]
@jsumners wdyt?

**Pull request merged**